### PR TITLE
1727 change button text and observe spelling

### LINKED ISSUE #1727: 🐛 [Bug] Change button text and observe spelling
## 🐛 Bugreport
The button for input (first page) should be called "Check input" in both cases, because "Generate now" or "Send now" is on the second page. 
Instead of "Reset", it would be better to say "Cancel" or "Clear form". (Edited)
Bildschirmfoto_2022-04-02_um_09 24 18

Instead of "Cancel" better "Back". 
"Generate Now" or "Send Now": Capitalise "Now". 
Bildschirmfoto_2022-04-02_um_09 28 32

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
button text change , button type from success to primary (gradido grün)

### Issues
- fixes #1727 

FireShot Capture 1078 -  - localhost
FireShot Capture 1078 -  - localhost
FireShot Capture 1079 -  - localhost
FireShot Capture 1080 -  - localhost
FireShot Capture 1081 -  - localhost

### CODE REVIEW

File: frontend/src/components/GddSend/TransactionConfirmationSend.spec.js [hunk 1]
```diff
@@ -63,8 +63,7 @@ describe('GddSend confirm', () => {
 
         describe('double click', () => {
           beforeEach(async () => {
-            await wrapper.find('button.btn-success').trigger('click')
-            await wrapper.find('button.btn-success').trigger('click')
+            await wrapper.find('button.btn-primary').trigger('click')
```
Comment (Mogge) [inline frontend/src/components/GddSend/TransactionConfirmationSend.spec.js:66]:
```suggestion
            await wrapper.find('button.btn-primary').trigger('click')
            await wrapper.find('button.btn-primary').trigger('click')            
```

**Review by Mogge — CHANGES_REQUESTED** [review]

**Comment by ulfgebhardt** [comment]
On Page 2 please use `Back` & `Zurück` instead of `Edit` & `Bearbeiten`

**Review by ulfgebhardt — DISMISSED** [review]
On Page 2 please use `Back` & `Zurück` instead of `Edit` & `Bearbeiten`

**Comment by ogerly** [comment]
FireShot Capture 1082 -  - localhost
FireShot Capture 1083 -  - localhost

**Review by Mogge — APPROVED** [review]

**Pull request merged**